python/adelphi/bin/adelphi

```diff
@@ -76,7 +105,7 @@ def export(ctx, hosts, port, username, password, keyspaces, rf, anonymize, outpu
     ctx.obj['port'] = port
     ctx.obj['username'] = username
     ctx.obj['password'] = password
-    ctx.obj['keyspace_names'] = keyspaces.split(',') if keyspaces is not None else None
+    ctx.obj['keyspace-names'] = keyspaces.split(',') if keyspaces is not None else None
```

Comment:
Part of an attempt throughout this PR to uniformly choose between underscores and hyphens when separating names.